UI/crash-report.cpp

```diff
@@ -20,7 +20,15 @@ OBSCrashReport::OBSCrashReport(QWidget *parent, const char *text)
 	textBox = new QPlainTextEdit;
 	textBox->setPlainText(QT_UTF8(text));
 	textBox->setLineWrapMode(QPlainTextEdit::NoWrap);
-	textBox->setFont(QFontDatabase::systemFont(QFontDatabase::FixedFont));
+
+	const QFont fixedFont =
+		QFontDatabase::systemFont(QFontDatabase::FixedFont);
+	textBox->setFont(fixedFont);
```

Comment:
Is this line actually doing anything? IIRC Qt vaguely tells people not to use `setFont` on Widgets otherwise styled via stylesheets.

Reply:
Removed.